### PR TITLE
[1LP][RFR] Removed VersionPick for unsupported versions (part 2)

### PR DESCRIPTION
**Pull request by ohiliazo**
__Updating tests__ to remove support of 5.7 version.

Removed unnecessary VersionPick from:
cfme/containers/container.py
cfme/services/myservice/ssui.py
cfme/storage/manager.py
cfme/storage/volume.py

### CODE REVIEW

File: cfme/containers/container.py [hunk 1]
```diff
@@ -19,10 +18,7 @@
 
 class ContainerAllView(ContainerObjectAllBaseView):
     """Containers All view"""
-    summary = Text(VersionPick({
-        Version.lowest(): '//h3[normalize-space(.) = {}]'.format(quote('All Containers')),
-        '5.8': '//h1[normalize-space(.) = {}]'.format(quote('Containers'))
-    }))
+    summary = Text('//h1[normalize-space(.) = {}]'.format(quote('Containers')))
```
Comment (mfalesni) [inline cfme/containers/container.py]: Since this is a constant, can you put the text directly into the locator?

Comment (ohiliazo) [reply cfme/containers/container.py]: @mfalesni just for clearance, do you mean to put it here:
```    def is_displayed(self):
        return Text('//h1[normalize-space(.) = {}]'.format(quote('Containers'))).is_displayed
```

Comment (mfalesni) [reply cfme/containers/container.py]:
```python
summary = Text('//h1[normalize-space(.) = "Containers"]')
```

Comment (ohiliazo) [reply cfme/containers/container.py]: @mfalesni, done.

**Pull request merged**